pages/api/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface Message {
```

Comment:
In the front end we already have a Message type so can we name this one MessageTemplate or something